src/screens/wallet/wallet.tsx

```diff
@@ -65,6 +82,23 @@ export const Wallet = () => {
       <TokenList />
 
       <BackupYourWalletOverlay />
+
+      <BottomSheet
+        title="Add this address to Contacts?"
+        description={addContactRequest}
+        cancelButtonText="Now now"
+        contentHeight={formatSize(180)}
+        controller={bottomSheetController}
+        onClosePress={handleClosePress}
+      >
+        <BottomSheetActionButton
+          title="Add address"
+          onPress={() => {
+            navigate(ModalsEnum.AddContact, { name: '', publicKeyHash: addContactRequest });
```

Comment:
`addContactRequest` has a bad name, it very noticable here `publicKeyHash: addContactRequest`